pom.xml

```diff
@@ -409,11 +412,48 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>com.exasol</groupId>
+                <artifactId>error-code-crawler-maven-plugin</artifactId>
+                <version>0.1.1</version>
```

Comment:
update to 5.0.0.

Reply:
for example using the mavenDependencyUpdate script

Reply:
Created a ticket to add error codes: https://github.com/exasol/row-level-security/issues/103